src/main/java/htsjdk/samtools/util/AsciiWriter.java

```diff
@@ -28,6 +28,7 @@
 import java.io.IOException;
 import java.io.OutputStream;
 import java.io.Writer;
+import java.nio.charset.*;
```

Comment:
By convention we try to avoid import *, unless not doing so would require a really large list. In this case, it looks like you're only using `StandardCharsets`, so you can just import that.